server/chat-plugins/scavengers.ts

```diff
@@ -1593,7 +1592,6 @@ const ScavengerCommands: ChatCommands = {
 		) {
 			return this.errorReply("You cannot edit the hints and answers if you are not the host.");
 		}
-
```

Comment:
Why are you removing a line of whitespace?

Reply:
There was initially a change there that I have now removed. The whitespace was a residual effect of that change. If preferred, it can be put back